### PR TITLE
Disabilities metrics tests

### PR DESCRIPTION
**Pull request by MariaGarciaCS**
The tests for the disability rate metric all pass, but two other tests do not. 

```
pipeline/tests/database_test.py::test_table_counts FAILED
pipeline/tests/database_test.py::test_belonging FAILED 
```

I updated my repo following the thread in "#help" between Vinesh and Asude, but the tests continue to fail. The only file I modified is community_test.py

### CODE REVIEW

File: api/metrics/community_test.py [hunk 1]
```diff
@@ -210,4 +210,47 @@ def test_belonging():
...
+            "period_end_year": 2016,
+            "segment": "all",
+            "value": 11.02
+        },
+        {
+            "area_number": 3,
+            "period_end_year": 2016,
+            "segment": "all",
+            "value": 8.46
+        },
```
Comment (vingkan) [inline api/metrics/community_test.py:234]: Good test! I would add one more row with a segment that is not "all" to show that the segment filter works.

**Review by vingkan — CHANGES_REQUESTED** [review]
Thanks for adding this test! GitHub says your tests pass and I pulled your branch to run tests on my Cloud9 and they pass there as well.

I suggested one change to the input data for your test. After you make that change you can merge.

**Comment by MariaGarciaCS** [comment]
Test for segment filter has been added. All tests pass.

**Review by vingkan — APPROVED** [review]
Good to merge! Thanks for updating the test.

Based on the diff, it seems like your PR changed the compressed database file even though it shouldn't have. The tests pass, so it's probably an safe change, and we don't have an easy way to fix that after it's been committed. So let's go ahead and merge it for now.

**Pull request merged**